### ISSUE TITLE
l0 task logs <taskname>/<taskid> outputs error message when there are no logs

### ISSUE THREAD

**Issue by sesh-kebab** [quintilesims/layer0#189]
```
$ l0 task logs <taskname>/<taskid> 
invalid character 'r' looking for beginning of value
```

This only happens when there are no logs for that task; so assuming it is a simple case of empty logs not being handled correctly.

**Comment by zpatrick** [comment]
Would like to tack on to this issue: 
* Fetching logs is very slow 
* The logs we get often aren't the latest logs. You can usually see newer logs in the cloudwatch console.

**Comment by diemonster** [comment]
More stuff to tack on:

* logs are consistently behind 'latest' logs delivered from AWS UI and or `aws` cli. @tlake reports up to 14 minutes behind, in fact. I'm sure we're doing something wrong re: getting the latest logs via the backend.

**Comment by sesh-kebab** [comment]
Fixed by #241
